developers_chamber/scripts/__init__.py

```diff
@@ -14,8 +14,10 @@ def find_and_replace_command_variable(arg, command):
     match = re.match(r'^--(?P<arg_name>[^=\ ]+)[\ =](?P<arg_value>.+)', arg)
     if match:
         arg_name, arg_value = match.groups()
-        if '$({})'.format(arg_name) in command:
-            return True, command.replace('$({})'.format(arg_name), arg_value)
+        if '${}'.format(arg_name) in command:
+            return True, command.replace('${}'.format(arg_name), arg_value)
+        elif '${}'.format(arg_name.replace('-', '_')) in command:
+            return True, command.replace('${}'.format(arg_name.replace('-', '_')), arg_value)
```

Comment:
there was typo not $(FOO) but $FOO is required. BASH doesn't support variable with - but we want to send parameters with - to the pydev